### ISSUE TITLE
Feature Request: 

### ISSUE THREAD

**Issue by flecmart** [grocy/grocy#310]
Recurring diet plan:

Hey - I am looking for a way to automate a diet plan. I found grocy a very interesting project and I saw in the demo that you can maintain recipes and assign them to days on a diet plan.  It would be awesome to be able to define a recurring diet plan that repeats after e.g. 6 weeks.

Thank you for the good work and best regards.

**Comment by berrnd** [comment]
Great idea.

So would be generally speaking about "meal plan templates" or "recurring meal plans"...

**Comment by flecmart** [comment]
Yes, I am searching for a possibilty to implement sth like this (https://www.konsumguerilla.net/der-ewige-speiseplan/) in our familiy. I didn't find an english version of it so sorry to the non german speakers ;-)

So erstellst Du Deinen persönlichen ewigen Speiseplan :

1. Brainstorming – Alle Gerichte, die ihr regelmäßig esst, auflisten
2. Wochenplan erstellen – Nudeln, Reis, Kartoffeln, Fleisch etc. feste Tage zuweisen
3. Frequenz festlegen – Wie oft soll in den 12 Wochen welches Gericht gekocht werden? Lieblingsgericht häufiger, als unbeliebte Speisen.
4. Lücken füllen – Du brauchst 84 Positionen um die 12 Wochen zu füllen. Plane neue Rezepte und Essen Gehen mit ein.
5. Plan aufstellen – Verteile die 84 Gerichte auf die 12 Wochen. Bedenke dabei, dass frische Zutaten rasch verarbeitet werden müssen. Berücksichtige also auch, wann Du jeweils einkaufen möchtest und wie lange sich etwa frisches Gemüse hält.

Combined with grocy's grocery list and monitoring capabilities this could be an awesome solution.
